content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -214,6 +214,11 @@ $ sed -e 's/from: Same/from: All/'\
 ' @samples/bookinfo/gateway-api/bookinfo-gateway.yaml@ | kubectl apply -f -
 {{< /text >}}
 
+{{< warning >}}
+Kindly make sure you have a IP address for the bookinfo-gateway service in Kind, you can install [MetalLB](https://kind.sigs.k8s.io/docs/user/loadbalancer/)
```

Comment:
Should we add this warning in kind platform setup https://istio.io/latest/docs/setup/platform-setup/kind/ and refer to the setup somewhere? Since it's more related to gateway-api.

Reply:
Yes that's a good suggestion, can we also keep it here? Because if I go to the ambient guide directly it's not clear.

I'll update and push shortly.

Reply:
Hi @hanxiaop should I rebase?